Library/Formula/jsawk.rb

```diff
@@ -1,16 +1,18 @@
-require 'formula'
-
 class Jsawk < Formula
-  homepage 'https://github.com/micha/jsawk'
-  url 'https://github.com/micha/jsawk/archive/1.4.tar.gz'
-  sha1 '4f2c962c8a5209764116457682985854400cbf24'
+  homepage "https://github.com/micha/jsawk"
+  url "https://github.com/micha/jsawk/archive/1.4.tar.gz"
+  sha1 "4f2c962c8a5209764116457682985854400cbf24"
 
-  head 'https://github.com/micha/jsawk.git'
+  head "https://github.com/micha/jsawk.git"
 
-  depends_on 'spidermonkey'
+  depends_on "spidermonkey"
 
   def install
-    mv "README.markdown", "README"
```

Comment:
I think a good standard is what ext files will be picked up by Github to show in the repo's front page. But I haven't found any related document yet.